packages/patternfly-3/react-console/src/DesktopViewer/DesktopViewer.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Row, Col } from 'patternfly-react';
+
+import ManualConnection from './ManualConnection';
+import ConnectWithRemoteViewer from './ConnectWithRemoteViewer';
+import consoleDetailPropType from './consoleDetailPropType';
+
+const DesktopViewer = ({ children, spice, vnc, onGenerate, onDownload, ...props }) => (
+  <Grid fluid className="desktop-viewer-pf">
```

Comment:
nit, can we allow additional classes here? i.e. using `classnames` like we do elsewhere (allowing consumer to add additional class for the `Grid`)?

[example](https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-3/patternfly-react/src/components/Table/TableSelectionHeading.js#L9)

Reply:
Sure, done